Python/Product/VSInterpreters/Interpreter/InterpreterOptionsService.cs

```diff
@@ -344,9 +344,9 @@ sealed class InterpreterOptionsService : IInterpreterOptionsService {
             }
 
             return _packageManagerProviders.SelectMany(p => p.Value.GetPackageManagers(factory))
+                .Where(p => p != null)
                 .GroupBy(p => p.UniqueKey)
                 .Select(g => g.OrderBy(p => p.Priority).FirstOrDefault())
-                .Where(p => p != null)
```

Comment:
GetPackageManagers can return null so added the new where statement above so it filters out all null.

but i'm not sure if I need the  old null check still

Reply:
I think now the old code was correct

Reply:
I think by filtering the nulls first  the FirstOrDefault() call will always return something.